diff --git a/instrucoes.md b/instrucoes.md
index b5df9f4..5009fb3 100644
--- a/instrucoes.md
+++ b/instrucoes.md
@@ -9,6 +9,7 @@
 - [REACT] componente com estado [OK]
 - [REACT] comunbicação entre componentes (direta) [OK]
 - [REACT] comunbicação entre componentes (indireta) [OK]
-- [REACT] renderização condicional []
+
+- [REACT] renderização condicional [OK]
 - [REACT] renderização de listas []
   
\ No newline at end of file
diff --git a/src/components/condicional/Faq.tsx b/src/components/condicional/Faq.tsx
index 5198493..eb54a68 100644
--- a/src/components/condicional/Faq.tsx
+++ b/src/components/condicional/Faq.tsx
@@ -1,12 +1,49 @@
+import { useState } from "react";
 import Pergunta from "./Pergunta";
 
 export default function Faq() {
+    const [ativo, setAtivo] = useState<number>(0)
+
+    function alternarVisibilidade(indice: number) {
+        if (indice === ativo) {
+            setAtivo(-1)
+        } else {
+            setAtivo(indice)
+        }
+    }
+
+    
+
     return (
         <div className="flex flex-col gap-2 w-[90%] md:w-3/4">
-            <Pergunta texto="Com quantos mamões se faz uma mamada?" resposta="The standard Lorem Ipsum passage, used since the 1500s Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat." />
-            <Pergunta texto="Se eu chupo uma laranja em 10 minutos quanto tepo levo pra chupar um saco?" resposta="The standard Lorem Ipsum passage, used since the 1500s Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat." />
-            <Pergunta texto="Se os marcianos são de marte, então os da terra são terraplanistas?" resposta="The standard Lorem Ipsum passage, used since the 1500s Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat." />
-            <Pergunta texto="Se arroz doce é doce, então o arroz normal é salgado?" resposta="The standard Lorem Ipsum passage, used since the 1500s Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat." />
+            <Pergunta
+                indice={0}
+                aberta={ativo === 0}
+                texto="Com quantos mamões se faz uma mamada?" 
+                resposta="The standard Lorem Ipsum passage, used since the 1500s Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
+                alternarVisibilidade={alternarVisibilidade}
+            />
+            <Pergunta
+                indice={1}
+                aberta={ativo === 1}
+                texto="Se eu chupo uma laranja em 10 minutos quanto tempo levo pra chupar um saco?" 
+                resposta="The standard Lorem Ipsum passage, used since the 1500s Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat." 
+                alternarVisibilidade={alternarVisibilidade}
+            />
+            <Pergunta
+                indice={2} 
+                aberta={ativo === 2}
+                texto="Se os marcianos são de marte, então os da terra são terraplanistas?"
+                resposta="The standard Lorem Ipsum passage, used since the 1500s Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
+                alternarVisibilidade={alternarVisibilidade}
+            />
+            <Pergunta
+                indice={3}
+                aberta={ativo === 3}
+                texto="Se arroz doce é doce, então o arroz normal é salgado?" 
+                resposta="The standard Lorem Ipsum passage, used since the 1500s Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
+                alternarVisibilidade={alternarVisibilidade}
+            />
         </div>
     )
 }
\ No newline at end of file
diff --git a/src/components/condicional/Pergunta.tsx b/src/components/condicional/Pergunta.tsx
index 195e585..2a5d47b 100644
--- a/src/components/condicional/Pergunta.tsx
+++ b/src/components/condicional/Pergunta.tsx
@@ -1,27 +1,30 @@
 import If from "@/components/condicional/If"
 import {IconChevronDown, IconChevronUp } from "@tabler/icons-react"
-import { useState } from "react"
+
 
 interface PerguntaProps {
+    indice: number
     texto: string
-    resposta :string 
+    resposta :string
+    aberta: boolean
+    alternarVisibilidade: (indice: number) => void
 }
 export default function Pergunta(props: PerguntaProps) {
-    const [aberta, setAberta] = useState<boolean>(false)
-
     return (
         <div className={`
             border border-zinc-600 rounded-md overflow-hidden
         
         `}>
             <div 
-                className="bg-zinc-800 p-5 cursor-pointer select-none flex justify-between"
-                onClick={() => setAberta(!aberta)}
-            >
+                className="bg-zinc-800 p-5 cursor-pointer select-none flex justify-between" onClick={() => props.alternarVisibilidade(props.indice)}>
                 <span>{props.texto}</span>
-                {aberta ? <IconChevronDown /> : <IconChevronUp />}
+                {props.aberta ? <IconChevronDown /> : <IconChevronUp />}
+                {/*<IfElse teste={aberta}>
+                    <IconChevronUp />
+                    <IconChevronDown />
+                </IfElse>*/}
             </div>
-            <If teste={aberta}>
+            <If teste={props.aberta}>
                 <div className="text-green-500 p-5">{props.resposta}</div>
             </If>
         
